app/server/endpoint/src/test/java/io/syndesis/server/endpoint/v1/handler/meta/AggregateMetadataHandlerTest.java

```diff
@@ -90,22 +90,7 @@ public void shouldExtractJavaInputElementVariant() throws IOException {
                                 .build())
                         .addVariant(dummyShape(DataShapeKinds.JSON_INSTANCE))
                         .build())
-                .outputShape(new DataShape.Builder()
-                        .kind(DataShapeKinds.JAVA)
-                        .specification(getSpecification("person-spec.json"))
-                        .description("person")
-                        .type(Person.class.getName())
-                        .putMetadata(StepMetadataHandler.VARIANT_METADATA_KEY, StepMetadataHandler.VARIANT_ELEMENT)
-                        .addVariant(new DataShape.Builder()
-                                .kind(DataShapeKinds.JAVA)
-                                .specification(getSpecification("person-list-spec.json"))
-                                .collectionType("List")
-                                .type(Person.class.getName())
-                                .collectionClassName(List.class.getName())
-                                .putMetadata(StepMetadataHandler.VARIANT_METADATA_KEY, StepMetadataHandler.VARIANT_COLLECTION)
-                                .build())
-                        .addVariant(dummyShape(DataShapeKinds.JSON_INSTANCE))
-                        .build())
+                .outputShape(StepActionHandler.NO_SHAPE)
```

Comment:
We lost a test case for `JAVA` data shape kinds here, should this be a new test case for `NO_SHAPE` instead?

Reply:
The test was an actual duplicate with the next test variant where the exact same output shape was used and verified. I changed that so we have input and output shape tested in separate tests